MaterialLib/MPL/CreateProperty.cpp

```diff
@@ -113,6 +113,11 @@ std::unique_ptr<MaterialPropertyLib::Property> createProperty(
         return createSaturationVanGenuchten(config);
     }
 
+    if (boost::iequals(property_type, "CapillaryPressureVanGenuchten"))
+    {
+        return createSaturationVanGenuchten(config);
```

Comment:
Doesn't match. CapillaryPressure property is created, but saturation is returned.

Reply:
Made it be inherited from MPL::Property.